### PR TITLE
improves raider outlaw loadouts a bit and adds a few new ones

### PR DESCRIPTION
**Pull request by chemistrymain3**
adds  a couple new outlaw loadouts

improves a bunch of other outlaw loadouts, scaling their power up considerably (but not stupidly, hopefully)

removes the skillbook picking thing, instead skillbooks come with the outlaw.

all of these are up for review tbh

anyway this also coincidentally buffs the autopipe rifle a bit

if it's too strong i'll nerf it

this was requested by a mob of screamig ppl on discord

## Why It's Good For The Game

current outlaw loadouts are piss awful

the new ones should be more manageable, tbh. nobody has guns that are much stronger than like, magneto wattzes or so forth. some of these loadouts are stronger than others but yah

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
tweak: improves outlaw loadouts a bit
tweak: buffs the autopipe rifl a lil
add: some new outlaw loadouts
/:cl:

### CODE REVIEW

**Review by zuliouse — APPROVED** [review]

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -478,70 +479,100 @@ Raider
...
 	backpack_contents = list(
-		/obj/item/gun/ballistic/automatic/pistol/pistol22 = 1,
-		/obj/item/ammo_box/magazine/m22,
-		/obj/item/reagent_containers/hypospray/medipen/psycho=1,
-		/obj/item/reagent_containers/inhaler/turbo=1)
+		/obj/item/gun/energy/laser/wattz/magneto = 1,
+		/obj/item/stock_parts/cell/ammo/ec = 2,
+		/obj/item/reagent_containers/hypospray/medipen/psycho = 3,
+		/obj/item/reagent_containers/inhaler/turbo = 2,
+		/obj/item/reagent_containers/syringe/medx = 1,
```
Comment (zuliouse) [inline code/modules/jobs/job_types/wasteland.dm:496]: /obj/item/reagent_containers/inhaler/jet
/obj/item/reagent_containers/hypospray/medipen/medx

**Review by zuliouse — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/wasteland.dm [hunk 2]
```diff
@@ -478,70 +479,100 @@ Raider
...
 
+/datum/outfit/loadout/quack_doctor
+	name = "Quack Doctor"
+	suit = /obj/item/clothing/suit/toggle/labcoat/f13/followers
+	l_hand = /obj/item/storage/backpack/duffelbag/med/surgery
+	r_hand = /obj/item/book/granter/trait/medical
+	suit_store = /obj/item/gun/energy/laser/wattz
+	backpack_contents = list(
+		/obj/item/stock_parts/cell/ammo/ec=1,
+		/obj/item/reagent_containers/pill/patch/jet = 3,
```
Comment (zuliouse) [inline code/modules/jobs/job_types/wasteland.dm:520]: /obj/item/reagent_containers/inhaler/jet

Comment (AdamElTablawy) [reply code/modules/jobs/job_types/wasteland.dm]: resolve

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
Comment (AdamElTablawy) [reply code/modules/jobs/job_types/wasteland.dm]: resolve

**Comment by Pebblesthe2nd** [comment]
Stumbled on this PR and saw its using assets from the light/medium/heavy armor rework that I was doing pre-great revolution. Its not really meant to coexist with the old system, and will create some issues with redundancies down the line. 

Unless you're adopting and finishing the system it might be advisable to just untick it or delete it all together, instead of modifying individual items to fit with linearmor again. This item is the one I refer to:
/obj/item/clothing/suit/armored/light/duster/desperado

**Comment by AdamElTablawy** [comment]
> Stumbled on this PR and saw its using assets from the light/medium/heavy armor rework that I was doing pre-great revolution. Its not really meant to coexist with the old system, and will create some issues with redundancies down the line.
> 
> Unless you're adopting and finishing the system it might be advisable to just untick it or delete it all together, instead of modifying individual items to fit with linearmor again. This item is the one I refer to: /obj/item/clothing/suit/armored/light/duster/desperado

We're finishing the system ourself, yeah.

**Review by zuliouse — APPROVED** [review]

**Pull request merged**